tests/Microsoft.Identity.Test.Android.UIAutomation/androidTests.cs

```diff
@@ -95,8 +95,7 @@ public void RunAllTests()
                 B2CFacebookMicrosoftAuthorityAcquireTokenTest,
                 //B2CGoogleB2CLoginAuthorityAcquireTokenTest,
                 //B2CGoogleMicrosoftAuthorityAcquireTokenTest,
-                B2CLocalAccountAcquireTokenTest,
-                B2CFacebookEditPolicyAcquireTokenTest
+                B2CLocalAccountAcquireTokenTest
```

Comment:
why is the edit profile policy test removed?

Reply:
It is starting to fail randomly again. It is already disabled in MSAL V3 so I am just making V2 the same

Reply:
have we looked into why it's failing? can you make a GitHub issue to track this? this test also tests Prompt.None, so it's nice to have.

Reply:
We were looking at this a few weeks back but I dont think we ever got to the bottom of it. On MSAL V3, It was then removed from the appcenter tests with the intention of converting it to a desktop test i believe. ill check it out and file a bug if necessary.

Reply:
Yes, please log bugs. Otherwise we'll end up with tests like OboFlowTests